lib/helpers/deployments.rb

```diff
@@ -1,8 +1,6 @@
-require 'plugins/mcollective'
-require 'plugins/chatops'
-
 module Deployments # rubocop:disable Style/Documentation
   def deploy(branch, deleted)
+    require 'plugins/mcollective' if settings.use_mcollective
```

Comment:
Just below you also have a `if setings.use_mcollective`. Perhaps move it in that block?